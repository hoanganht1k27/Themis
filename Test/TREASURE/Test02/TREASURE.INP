16 12
20 52 62 6 37 29 66 72 93 78 70 63 83 94 49 95
12 4
5 2
9 14
7 12
5 11
8 9
5 9
7 3
2 12
8 6
10 9
16 12
2 4 8 7 11 6 1 3 12 9 5 10
