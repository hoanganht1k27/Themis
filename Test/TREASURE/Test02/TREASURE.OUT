782
720
654
617
539
446
394
300
205
142
72
0
